core/kernel.rbs

```diff
@@ -112,50 +113,50 @@ module Kernel : BasicObject
   # Array(nil)         #=> []
   # Array(1)           #=> [1]
   # ```
-  def Array: (NilClass x) -> [ ]
+  def self?.Array: (NilClass x) -> [ ]
            | [T] (::Array[T] x) -> ::Array[T]
            | [T] (::Range[T] x) -> ::Array[T]
            | [K, V] (::Hash[K, V] x) -> ::Array[[K, V]]
            | [T] (T x) -> ::Array[T]
 
-  def Complex: (Numeric | String x, ?Numeric | String y, ?exception: bool exception) -> Complex
+  def self?.Complex: (Numeric | String x, ?Numeric | String y, ?exception: bool exception) -> Complex
 
-  def Float: (Numeric | String x, ?exception: bool exception) -> Float
+  def self?.Float: (Numeric | String x, ?exception: bool exception) -> Float
 
-  def Hash: [K, V] (Object x) -> ::Hash[K, V]
+  def self?.Hash: [K, V] (Object x) -> ::Hash[K, V]
 
-  def Integer: (Numeric | String arg, ?exception: bool exception) -> Integer
+  def self?.Integer: (Numeric | String arg, ?exception: bool exception) -> Integer
              | (String arg, ?Integer base, ?exception: bool exception) -> Integer
 
-  def Rational: (Numeric | String | Object x, ?Numeric | String y, ?exception: bool exception) -> Rational
+  def self?.Rational: (Numeric | String | Object x, ?Numeric | String y, ?exception: bool exception) -> Rational
```

Comment:
> __[Goodcheck]__ Prefer `boolish` over `bool` for method arguments and block return values ([view](https://sider.review/gh/repos/174796313/pulls/549#issue-5268998))
>
> <details>
>
> See the doc below:
> https://github.com/ruby/rbs/blob/78d04a2db0f1c4925d2b13c2939868edf9465d6c/docs/syntax.md#bool-or-boolish
>
> | Rule |
> | ---- |
> | `rbs.prefer_boolish` |
>
> You can [close this issue](https://sider.review/gh/repos/174796313/pulls/549#issue-5268998) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I closed this issue, because:
- When you strictly want `true | false`.